lib/matplotlib/quiver.py

```diff
@@ -1009,7 +1009,8 @@ def _make_barbs(self, u, v, nflags, nbarbs, half_barb, empty_flag, length,
 
         *pivot* specifies the point on the barb around which the
         entire barb should be rotated.  Right now, valid options are
-        'head' and 'middle'.
+        'tip' and 'middle'. Can also be a number, which shifts the start
+        of the barb that many units from the origin.
```

Comment:
It looks like the default pivot uses [some factor of `length`](https://github.com/matplotlib/matplotlib/blob/master/lib/matplotlib/quiver.py#L1050) and `length` is [defined in points](https://github.com/matplotlib/matplotlib/blob/master/lib/matplotlib/quiver.py#L1008), so I guess this _should_ be points.

BTW, in looking through this, I found a typo in the docs; do you mind fixing it here?

This section says the [default length is 9](https://github.com/matplotlib/matplotlib/blob/master/lib/matplotlib/quiver.py#L783-L786], but the [default appears to be 7](https://github.com/matplotlib/matplotlib/blob/master/lib/matplotlib/quiver.py#L916).